py-polars/polars/series/datetime.py

```diff
@@ -154,15 +154,19 @@ def max(self) -> dt.date | dt.datetime | dt.timedelta | None:
         """
         return wrap_s(self._s).max()  # type: ignore[return-value]
 
+    @deprecate_function("Use `Series.median` instead.", version="0.20.32")
```

Comment:
Version should be 1.0.0 (please update the other mentions in Series and the docstrings as well):

```suggestion
    @deprecate_function("Use `Series.median` instead.", version="1.0.0")
```

Reply:
Done.